CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project are documented in this file.
 
 The format is *loosely* based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) .The project does *not* follow Semantic Versioning and the changes are documented in reverse chronological order, grouped by calendar month.
 
+## May 2025
```

Comment:
I guess you did not re-generate this file? 
To do it as intended you need to update some model content and re-run the gradle task as mentioned in the readme: https://github.com/JetBrains/MPS-extensions?tab=readme-ov-file#creating-a-pr

Otherwise your change log update will be overriden next time...